task/kamel-run/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- squakez
+- oscerd
+reviewers:
```

Comment:
can you please raise a PR in tektoncd/community to get added yourself into the community so that you can maintain this task ? example: https://github.com/tektoncd/community/pull/869

Reply:
Done in https://github.com/tektoncd/community/pull/1022